### PR TITLE
Use nightly compiler, support building C code

### PR DESCRIPTION
**Pull request by arturkow2000**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,15 +26,20 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o /tmp/rustup-ini
     chmod +x /tmp/rustup-init && \
     /tmp/rustup-init -y --no-modify-path \
         --default-toolchain nightly-2022-01-13 \
-        -t thumbv7em-none-eabihf && \
+        -t thumbv7em-none-eabihf \
+	-c llvm-tools-preview \
```
Comment (0xDen) [inline Dockerfile]: If `-c llvm-tools-prevew` is the option for `/tmp/rustup-init` then could be aligned with others?

Comment (arturkow2000) [reply Dockerfile]: Fixed.

Comment (krystian-hebel) [reply Dockerfile]: Not fixed, this is not in the same column as `-t ...` above.

**Review by krystian-hebel — APPROVED** [review]
Docker image is able to build and run both nRF and PC code with this change.

**Review by macpijan — APPROVED** [review]

**Pull request merged**